config/autoload/local.php.dist

```diff
@@ -12,6 +12,15 @@
  */
 
 return [
+    /**
+     * Proxy configuration, used when determinining user roles.
+     */
+    'proxy' => [
+        'enabled' => false,
+        'ip_addresses' => [],
+        'header' => 'X-Forwarded-For',
```

Comment:
Looks good, as long as you change this to `X-Real-Ip`

Reply:
Changed it!